src/Dockerfile.template

```diff
@@ -6,8 +6,8 @@
 {%     set ns.extra_pkgs = item.value %}
 {%   endif %}
 {% endfor %}
-{% if config.os.id == 'rhel' and spec.img_version %}
-{%   set img_tag = spec.img_version %}
+{% if config.os.id == 'rhel' and spec.base_img_version %}
```

Comment:
If the base_img_version is only used for rhel, how about `rhel_base_img_version` to make it clearer? Sorry for nitpicking, I just hate to debug these little things so I like to avoid them if possible.